creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1022,11 +1086,14 @@ class SnapbackSM {
       // Retrieve clock statuses for all users and their current replica sets
       let replicaSetNodesToUserClockStatusesMap
       try {
-        replicaSetNodesToUserClockStatusesMap =
-          await this.retrieveClockStatusesForUsersAcrossReplicaSet(
-            replicaSetNodesToUserWalletsMap,
-            unhealthyPeers
-          )
+        const {
+          replicasToUserClockStatusMap,
+          unhealthyPeers: clockUnhealthyPeers
+        } = await this.retrieveClockStatusesForUsersAcrossReplicaSet(
+          replicaSetNodesToUserWalletsMap
+        )
+        replicaSetNodesToUserClockStatusesMap = replicasToUserClockStatusMap
```

Comment:
we can just do this assignment on line 1090 right

Reply:
are you talking about something like this:
```js
let replicaSetNodesToUserClockStatusesMap, unhealthyPeersFromClockStatusRetrieval
try {
  ;({
    replicasToUserClockStatusMap: replicaSetNodesToUserClockStatusesMap,
    unhealthyPeers: unhealthyPeersFromClockStatusRetrieval
  } = await this.retrieveClockStatusesForUsersAcrossReplicaSet(
    replicaSetNodesToUserWalletsMap
  ))
  unhealthyPeers = new Set([...unhealthyPeers, ...unhealthyPeersFromClockStatusRetrieval])
} catch (e) {...}
```

this works but I don't like how:
1. it has to declare `unhealthyPeersFromClockStatusRetrieval` above the try-catch, but we never need that variable outside the try so it's polluting the scope
2. imo the `;({variable: existingVariable}) = whatever()` expression is harder to read than just `const {variable} = whatever(); existingVariable = variable`

wdyt?

Reply:
agreed, i dont like that `'({})` notation either
how about for consistency we do
```
const clockStatusResp = await this.retrieveClockStatusesForUsersAcrossReplicaSet(replicaSetNodesToUserWalletsMap)
const replicaSetNodesToUserClockStatusesMap = clockStatusresp.replicasToUserClockStatusMap
unhealthyPeers = new Set([...unhealthyPeers, ...clockStatusResp.unhealthyPeers])
```

Reply:
anyway, nbd - your call

Reply:
perfect. pushing that now